### DISCUSSION TITLE
Sequins feature random CV generation

### DISCUSSION THREAD

**Discussion by Fibweb** [djphazer/O_C-Phazerville#60] [Ideas]
One feature of Sequins would be very nice. The possiblity to create random CV Values for the Sequence, like shredder. 
Unfortunately, it is not possible to save the generated CV sequences in the shredder.

**Comment by djphazer** [comment]
This could be a simple feature to implement, but I'm not personally planning to rework or augment the original apps (like Sequins) ... though they might get some attention from the original developers at some point.

I am, however, working on a new prescriptive sequencer applet in Hemisphere called **Seq32** - with some randomization functions... with the sequences being stored in the same banks that Sequins uses, but in a different format.

**Comment by Fibweb** [comment]
thanks for the quick reply. And thanks in general for your brilliant work on the firmware
